asab/application.py

```diff
@@ -773,9 +773,10 @@ def _on_housekeeping_tick(self, message_type):
 
 		if self.HousekeepingTime < now:
 			if now < self.HousekeepingTimeLimit and self.HousekeepingId <= today_id:
+				L.log(asab.LOG_NOTICE, "is performing housekeeping.")
```

Comment:
i suggest making it a full sentence (such as `Housekeeping started.`) which does not rely on the module name `asab.application` being there. should we parse the message into a structured event, it would get separated anyway i guess.

Reply:
Alright. I was inspired by `asab.application is ready.`...